website_docs/manual.md

```diff
@@ -194,6 +194,50 @@ Events can also have attributes of their own -
 span.AddEvent("Cancelled wait due to external signal", trace.WithAttributes(attribute.Int("pid", 4328), attribute.String("signal", "SIGHUP")))
 ```
 
+### Set span status
+
+A status can be set on a span, typically used to specify that there was an error in the operation a span is tracking - .`Error`.
+
+```go
+import (
+	// ...
+	"go.opentelemetry.io/otel/codes"
+	// ...
+)
+
+// ...
+
+result, err := operationThatCouldFail()
+if err != nil {
+	span.SetStatus(codes.Error, "operationThatCouldFail failed")
+}
+```
+
+By default, the status for all spans is `Unset`. In rare cases, you may also wish to set the status to `Ok`. This should generally not be necessary, though.
+
+### Record errors
+
+If you have an operation that failed and you wish to capture the error it produced, you can record that error.
+
+```go
+import (
+	// ...
+	"go.opentelemetry.io/otel/codes"
+	// ...
+)
+
+// ...
+
+result, err := operationThatCouldFail()
+if err != nil {
+	span.SetStatus(codes.Error, "operationThatCouldFail failed")
```

Comment:
https://opentelemetry.io/docs/instrumentation/go/getting-started/

1. setting the status before recording an error - This differs from the "Getting Started" documentation.
2. Similarly, why use `"operationThatCouldFail failed"` for the description as opposed to `err.Error()`. Is there a best practice?